samples/batch/transcription-enabled-storage/Connector/EnvironmentVariables/CreateTranscriptionEnvironmentVariables.cs

```diff
@@ -0,0 +1,58 @@
+﻿// <copyright file="CreateTranscriptionEnvironmentVariables.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector
+{
+    using System;
+
+    public static class CreateTranscriptionEnvironmentVariables
```

Comment:
Does it really create environment variables?

Reply:
We need environment variables to get the configuration values when setting up the function in the azure portal (via ARM or manually).

You can use keyvault for that, but the code would look the same - it's just that the env var is set differently:
https://docs.microsoft.com/en-gb/azure/app-service/app-service-key-vault-references